content/docs/hooks-effect.md

```diff
@@ -133,21 +133,21 @@ function Example() {
   });
 ```
 
-We declare the `count` state variable, and then we tell React we need to use an effect. We pass a function to the `useEffect` Hook. This function we pass *is* our effect. Inside our effect, we set the document title using the `document.title` browser API. We can read the latest `count` inside the effect because it's in the scope of our function. When React renders our component, it will remember the effect we used, and then run our effect after updating the DOM. This happens for every render, including the first one.
+ここでは `count` という state 変数を宣言し、次に副作用を使うことを React に伝えています。`useEffect` フックには関数を渡しています。この関数こそが副作用関数です。この副作用関数内で `document.title` というブラウザ API を使ってドキュメントのタイトルを設定しています。副作用関数は関数スコープ内にあるため最新の `count` の値は副作用内から参照可能です。React がコンポーネントをレンダーする際に React はこの副作用を覚えておき、DOM を更新した後に呼び出します。これは初回を含む毎回のレンダー時に発生します。
```

Comment:
```suggestion
ここでは `count` という state 変数を宣言し、次に副作用を使うことを React に伝えています。`useEffect` フックには関数を渡しています。このフックに渡された関数こそが副作用関数です。この副作用関数内で `document.title` というブラウザ API を使ってドキュメントのタイトルを設定しています。副作用関数は関数スコープ内にあるため最新の `count` の値は副作用内から参照可能です。React がコンポーネントをレンダーする際、React は実行された副作用を覚えており、DOM を更新した後にその副作用を呼び出します。これは初回を含む毎回のレンダー時に発生します。
```

Reply:
「実行された副作用を覚えており」はおかしいです、Reactは useEffect が呼ばれた時点では副作用は単に（後の実行のために）その副作用関数を記憶しているだけであり実行はしていないからです

Reply:
たしかにそのとおりでした。ご指摘ありがとうございます。
これはレビューではなく相談というか疑問というかなんですが、`it will remember the effect we used` の `we used` ってどういう意味なんでしょう…？
直訳すると「私たちが使った」ですが、この関数はまだ使われてないですよね？

Reply:
we used は深く考えていませんでしたが「useEffectに渡すのに使った」というだけかなと…たぶん

Reply:
なるほど…俗語的な感じなのでしょうか…
これが謎だなと思っていました。
お答えいただきありがとうございます！